friends/src/components/Friends.js

```diff
@@ -1,19 +1,22 @@
 import React, { Component } from 'react';
 
 class Friends extends Component {
-    constructor(){
-    super();
-    this.state ={
-        friends: []
-    }
-    
-    
-    
-    return({
-
-    })
+    constructor(props){
+    super(props);
+   
+} 
+    render(){
+    return(
+        <div>
+            <p>Name:{this.props.friend.name}</p>
```

Comment:
Over here, you might want to deconstruct this.props.friend into 3 variables: name, age and email. Overall, it will reduce time it takes to type, also it will make the code easier to read.